src/util/require.ts

```diff
@@ -1,8 +1,9 @@
 import { createRequire } from 'node:module';
+import { join } from 'node:path';
 import { toPosix } from './path.js';
 import { timerify } from './performance.js';
 
-const require = createRequire(process.cwd());
+const require = createRequire(join(process.cwd(), 'package.json'));
```

Comment:
This doesn't seem safe. We probably need to actually look for the `package.json` file, and each workspace probably needs to refer to its own `package.json`.

Reply:
Made this change but it might be a bit of a hack.